docs/UserGuide.md

```diff
@@ -3,64 +3,65 @@ layout: page
 
-* Table of Contents
-{:toc}
+- Table of Contents
+  - [Quick Start](#quick-start)
 
---------------------------------------------------------------------------------------------------------------------
+---
 
-## Quick start
+## Quick Start
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
-
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
-
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
-
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
-   Some example commands you can try:
-
-   * `list` : Lists all contacts.
-
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
-
-   * `delete 3` : Deletes the 3rd contact shown in the current list.
-
-   * `clear` : Deletes all contacts.
-
-   * `exit` : Exits the app.
+2. Type the command in the command box.
+
+3. Press Enter to execute it. e.g. typing help and pressing Enter will open the help window.
+
+- List
+  - `list /module`: Lists the names of all the recorded modules.
+  - `list /module /{module_code}`: Lists all the videos recorded in that module code
+- Add
+  - `add-module {module_code} [/name {module_name}]`: Adds a module to Le Tracker
+  - `add-lecture /module {module_code}`: Adds a lecture to a module
+  - `add-video /module {module_name} /lecture {lecture_index} /video {video_name}`: Adds a video to the module code
+- Mark/Unmark
+  - `mark /module {module_code} /lecture {lecture_index} /video {video_name}`: Marks a video as watched
+  - `unmark /module {module_code} /lecture {lecture_index} /video {video_name}`: Unmarks a video as unwatched
+- Delete
+  - `delete-module {module_code}`: Deletes a module from Le Tracker
+  - `delete-lecture /module {module_code} /lecture {lecture_id}`: Deletes the specified lecture from the specified module
+  - `delete-video /module {module_code} /lecture {lecture_id} /video {video_id}`: Deletes the specified video from the specified lecture from the specified module
+- Tag
+  - `tag /module {module_code} /lecture {lecture_id} /description {tag_description}`: Tag a module from Le Tracker
+  - `untag /module {module_code} /lecture {lecture_id} /tag {tag_id}`: Untag a module from Le Tracker
```

Comment:
Think it'd be best to change "Tag a module ..." to "Tag a module ..." and also "Untag a module ..." to "Untags a module ...".

Since all the other descriptions are formatted with the first word ending with an  "s", probably would be good for standardisation purposes.